packages/ui/react-ui-dnd/src/components/ResizeHandle.tsx

```diff
@@ -65,52 +72,52 @@ export const ResizeHandle = ({
     onChange: onSizeChange,
   });
   const dragStartSize = useRef<Size>(size);
+
   const orientation = side.startsWith('inline') ? 'horizontal' : 'vertical';
   const client = orientation === 'horizontal' ? 'clientX' : 'clientY';
 
-  useLayoutEffect(
-    () => {
-      if (!buttonRef.current || buttonRef.current.hasAttribute('draggable')) {
-        return;
-      }
-      // TODO(thure): This should handle StackItem state vs local state better.
-      draggable({
-        element: buttonRef.current,
-        onGenerateDragPreview: ({ nativeSetDragImage }) => {
-          // We will be moving the line to indicate a drag; we can disable the native drag preview.
-          disableNativeDragPreview({ nativeSetDragImage });
-          // We don't want any native drop animation for when the user does not drop on a drop target.
-          // We want the drag to finish immediately.
-          preventUnhandled.start();
-        },
-        onDragStart: () => {
-          dragStartSize.current =
-            dragStartSize.current === 'min-content'
-              ? measureSubject(buttonRef.current!, fallbackSize)[orientation === 'horizontal' ? 'width' : 'height'] /
-                REM
-              : dragStartSize.current;
-        },
-        onDrag: ({ location }) => {
-          if (typeof dragStartSize.current !== 'number') {
-            return;
-          }
-          setSize(getNextSize(dragStartSize.current, location, client, minSize, side));
-        },
-        onDrop: ({ location }) => {
-          if (typeof dragStartSize.current !== 'number') {
-            return;
-          }
-          const nextSize = getNextSize(dragStartSize.current, location, client, minSize, side);
-          setSize(nextSize);
-          onSizeChange?.(nextSize, true);
-          dragStartSize.current = nextSize;
-        },
-      });
-    },
-    [
-      // Note that `size` should not be a dependency here since dragging this adjusts the size.
-    ],
-  );
+  useLayoutEffect(() => {
+    if (!buttonRef.current || buttonRef.current.hasAttribute('draggable')) {
```

Comment:
```suggestion
    if (!buttonRef.current) {
```
That predicate was an early misunderstanding that `return draggable` fixes.